tests/utils.py

```diff
@@ -151,6 +151,17 @@ def make_nifti_image(array, affine=None):
     return image_name
 
 
+def make_rand_affine(ndim: int = 3):
+    """Create random affine transformation (with values == -1, 0 or 1)."""
+    vals = np.random.choice([-1, 1], size=ndim)
```

Comment:
would be great to support random generator e.g.: https://github.com/Project-MONAI/MONAI/blob/master/monai/data/synthetic.py#L51-L55

Reply:
done!